packages/web/src/components/table/components/TablePlayButton.tsx

```diff
@@ -12,27 +13,42 @@ type TablePlayButtonProps = {
   onClick?: (e: any) => void
   paused?: boolean
   playing?: boolean
+  isTrackPremium?: boolean
+  isLocked?: boolean
 }
 
 export const TablePlayButton = ({
   className,
   hideDefault = true,
   onClick,
   paused,
-  playing = false
+  playing = false,
+  isTrackPremium = false,
+  isLocked = false
 }: TablePlayButtonProps) => {
+  const {
+    color: {
+      special,
+      primary: { p300 }
+    }
+  } = useTheme()
+  const showPremiumColor = isLocked && isTrackPremium
   return (
     <div onClick={onClick} className={cn(styles.tablePlayButton, className)}>
       {playing && !paused ? (
         <div>
-          <IconPause className={styles.icon} />
+          <IconPause
+            className={styles.icon}
+            fill={showPremiumColor ? special.lightGreen : p300}
+          />
         </div>
       ) : (
         <div>
           <IconPlay
             className={cn(styles.icon, {
               [styles.hideDefault]: hideDefault && !playing
             })}
+            fill={showPremiumColor ? special.lightGreen : p300}
```

Comment:
The `color` prop doesn't have a green one, thoughts on adding?

Reply:
`Button` has an override `hexColor` prop which we use for the purchase button. Maybe we want something similar here?